wurst/lib/PingMinimap.wurst

```diff
@@ -0,0 +1,52 @@
+package PingMinimap
+
+let DEFAULT_DURATION = 10.0
+
+public function pingMinimap(vec2 pos, real duration, color color, bool extraEffects)

```

Comment:
Is `extraEffects` only for the exclamation? I think an enum for the final parameter might make it more readable.